### PR TITLE
Have travis-ci upload an artifact to the github release when we tag commits

### PR DESCRIPTION
**Pull request by mcpherrinm**

### CODE REVIEW

**Comment by csstaub** [comment]
Can we have that for multiple platforms? How does it work?

File: .travis.yml [hunk 1]
```diff
@@ -22,3 +19,14 @@ before_script:
 script:
   - go test -v `glide nv`
   - PATH=$PWD:$PATH cram -v tests/*.t
+
+deploy:
+  provider: releases
+  api_key:
+    secure: i+82k3Khvke2ejCJ3B+BE0nsemPSXnwcm4cDtuC8BF3vXPSO36hIbPDZ+mnGlqiGw54tduY3jrytCwo1YHnxu8veT3cJ27p9wDQt7Lz6kLf1wRHBp180PL2bfz1tOugtDsAw9D78EJj0LvWtupY0CmzXl8idVua1T9Z5HTR4V28QoDw0JY/+GB+Aq+C51oyDcICtLJBVhc4Wl/K7AqZPWEBC/kn1fG6BYbFIHCNLho4qiCLW9aQdFL5usHJ07992VBlzXLSkP9WU1w/pZ1kT4YaopusyQH8FG1be+V8hf74N0I9vhR/YRTK4cOuAo3Gu87HiEK0U9pgubyoYTRWeZYS9ryVH8Wa7QelauWbH16OWeWv18w98/8aaKE7RgZHlrMFwXnzzFokXuFqzMNWKOP5QxMOqlkY6UcgwS7RqbjxvHyzfapb212s5qgN1LAE85fys2ExMFEPO2NLry9PZVD2bHbQ+Y6bQg6XiI8Kvvmz0dRaY+chADpDNjIK+zpDwc3o1MjvnwiUt7AZrj4PvVD381E0X3JWmMeOOaW46LC5hn5Bhobl6wDC1crgf3QAqfMhFlAd+O0j7qOrW31vm1wPMtaeNQIAq/xjGROvLeBgKlzx1bfK51YQxIC3wRA2AVrxispHkq+rK30lGPhNHWLb91qy4tuy7y1sUAnwm8Dk=
+  file: certigo
```
Comment (csstaub) [inline .travis.yml]: Should this be certigo.linux or something?

Comment (mcpherrinm) [inline .travis.yml]: Well, it's specifying the file from the build to upload.

**Comment by mcpherrinm** [comment]
changed to add mac, linux, & windows binaries (all cross-compiled from travis, which is linux)

**Comment by csstaub** [comment]
Since this just uploads the file, I'm not sure it'd be obvious that `certigo` is the linux/amd64 one. Can we make the names consistent, like `certigo-$GOOS-$GOARCH(.exe)`?

**Comment by mcpherrinm** [comment]
Looks better now?

**Pull request merged**